### ISSUE TITLE
NumberBoxBehavior - sample application

### ISSUE THREAD

**Issue by blahetal** [SuperJMN-Zafiro/Zafiro.Avalonia#8]
Hi, could you consider adding a sample application to demonstrate NumberBoxBehavior.
I tried to use it in my project but the textbox still accepted characters (not only numbers).
In debug mode only `OnAttachedToVisualTree` was called, the other methods to filter out characters were never called.

**Comment by SuperJMN** [comment]
Hi! 

I created the behavior just for fun, but it seems it still has some bugs.

I've just created a `TextBox` and it seems to work (with some bugs, it seems, because I have tried to enter a decimal number and a negative number and it didn't work).

The code I used is this:

```xml
	<TextBox Text="{Binding Greeting}" HorizontalAlignment="Center" VerticalAlignment="Center">
		<Interaction.Behaviors>
            <avalonia:NumberBoxBehavior />
        </Interaction.Behaviors>
	</TextBox>
```

**Comment by blahetal** [comment]
Yep, I am doing this but cannot get it working, could you attach your test project here please.

**Comment by SuperJMN** [comment]
Please, try with this:

[AvaloniaApplication1.zip](https://github.com/SuperJMN-Zafiro/Zafiro.Avalonia/files/9757185/AvaloniaApplication1.zip)

**Comment by SuperJMN** [comment]
> thanks, it helped

Glad it helped 👏
